input/system/selfhosted/system.go

```diff
@@ -38,8 +38,10 @@ func GetSystemState(server *state.Server, logger *util.Logger) (system state.Sys
 
 	statusBytes, err := exec.Command("/usr/bin/pganalyze-collector-helper", "status", config.DbDataDirectory).Output()
```

Comment:
Shouldn't we skip the whole helper invocation, since its not going to do anything useful on Crunchy Bridge?

Reply:
Yeah I was thinking about that, let me tweak a bit more